lib/app/list.ex

```diff
@@ -0,0 +1,53 @@
+defmodule App.List do
+  use Ecto.Schema
+  import Ecto.Changeset
+  import Ecto.Query
+  alias App.{Item, ListItem, Person, Repo}
+  alias __MODULE__
+
+  schema "lists" do
+    field :name, :string
+
+    belongs_to :people, Person, references: :person_id, foreign_key: :person_id
```

Comment:
`belongs_to` again the `Ecto` way of writing schemas ... 🙄 
It _really_ bugs me that we are forced to write this kind of non-standard ORM stuff just to create a lookup.
`lists` will be owned by `groups` but that should just be `list.group_id` unless we want to allow the _same_ list to belong to different `groups` in which case we need a lookup table.

Reply:
This `belongs_to` to the `person` will be removed/updated with the work on permission